### PR TITLE
Basic Functionalities of Stack

### PR DESCRIPTION
**Pull request by Alifbhanvadiya14**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Implemented Stack using List and added some function in Stack implementation using Linked List

### Checklist

- [✓] I've read the contribution guidelines.
- [✓] I've checked the issue list before deciding what to submit.
- [✓] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

-  Stack using List /Array was not implemented and some functionalities were missing in the implementation of Stack using Linked List.  So  I add these functionalities.

### CODE REVIEW

**Comment by Alifbhanvadiya14** [comment]
Review only basic functionalities of Stack files

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Alifbhanvadiya14 Please do following modifications
* Take dynamic input, i.e take input from the user.
* Create a count function which counts total number of elements present at that time.
* Display suitable message after each operation

File: Python/ds/Stacks.py [hunk 1]
```diff
@@ -39,25 +43,126 @@ def pop(self):
...
-	stack.push(50)
-	print ("Pass" if (stack.size() == 3) else "Fail")
\ No newline at end of file
+    ## Testing Stack implementation using Linked List
+
+    print("Representation using Linked List")
+
+    # Initialize stacks with some values
+    stack = Stack()
+    stack.push(10)
```
Comment (Kajol-Kumari) [inline Python/ds/Stacks.py:105]: Please take the input of the stack from user.

File: Python/ds/Stacks.py [hunk 2]
```diff
@@ -39,25 +43,126 @@ def pop(self):
...
+    print("Representation using List")
+    stack2 = StackList()
+    stack2.push(10)
+    stack2.push(20)
+    stack2.push(30)
+    stack2.push(40)
+    stack2.push(50)
+
+    # Test size()
+    print ("Pass" if (stack2.size() == 5) else "Fail")
```
Comment (Kajol-Kumari) [inline Python/ds/Stacks.py:149]: please take the size as input as well instead of hardcoding.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Alifbhanvadiya14 Please remove the hardcoded values and take the input from user. Also please add a sample input-output at the end of the code section in commented form.

**Pull request closed without merging**